tokio-postgres/src/lib.rs

```diff
@@ -241,14 +244,23 @@ pub enum AsyncMessage {
 
 /// Message returned by the `SimpleQuery` stream.
 #[non_exhaustive]
-#[derive(Debug)]
```

Comment:
Looks like we will need to keep this or we see a compile error in ReadySet--so we should derive Debug for CommandCompleteContents instead

Reply:
Done